java/src/jmri/jmrit/beantable/TurnoutTableAction.java

```diff
@@ -1204,7 +1204,7 @@ public void setMenuBar(BeanTableFrame f) {
         JMenuBar menuBar = f.getJMenuBar();
         // check for menu
         int menus = menuBar.getMenuCount();
-        if (menus < 4) {
+        if (menus < 3) {
```

Comment:
Works on OS X. Please check on Windows and Linux whether the set number of menu items is correct to build each menu (at least) once.